pkg/clients/imagebuilder/client.go

```diff
@@ -367,6 +380,16 @@ func (c *Client) ComposeInstaller(image *models.Image) (*models.Image, error) {
 				},
 			}},
 	}
+
+	customInstaller := &CustomInstaller{
+		Unattended:   true,
+		SudoNoPasswd: []string{"@wheel"},
+	}
+
+	if feature.BlueprintCustomInstaller.IsEnabled() {
+		req.Customizations.Installer = customInstaller
+	}
```

Comment:
I think we can inline it and that way not initiate when not needed :)

```suggestion
	if feature.BlueprintCustomInstaller.IsEnabled() {
		req.Customizations.Installer = &CustomInstaller{
			Unattended:   true,
			SudoNoPasswd: []string{"@wheel"},
		}
	}
```

Reply:
works for me. coming up...